Champions/Nidalee/Q-C.cs

```diff
@@ -0,0 +1,112 @@
+using System.Numerics;
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using LeagueSandbox.GameServer.API;
+using LeagueSandbox.GameServer.GameObjects;
+
+namespace Spells
+{
+    public class Takedown : IGameScript
+    {
+        public static Particle mark;

```

Comment:
Isn't there an interface for `Particle`?

Reply:
IDK, just look at the example from others spell, because currently On-Hit spell doesn't use that often